### PR TITLE
Sanitize SQL generated by SqlServerSequenceValueGenerator

### PR DESCRIPTION
**Pull request by mikary**
Using the ```ISqlServerSqlGenerator``` to sanitize the sequence name.
@ajcvickers @bricelam

### CODE REVIEW

**Comment by bricelam** [comment]
:shipit:

File: src/EntityFramework.SqlServer/SqlServerSequenceValueGenerator.cs [hunk 1]
```diff
@@ -13,28 +13,30 @@ namespace Microsoft.Data.Entity.SqlServer
...
             _executor = executor;
+            _sqlGenerator = generator;
             _connection = connection;
         }
 
         protected override long GetNewLowValue()
         {
-            // TODO: Parameterize query and/or delimit identifier without using SqlServerMigrationOperationSqlGenerator
-            var sql = string.Format(CultureInfo.InvariantCulture, "SELECT NEXT VALUE FOR {0}", _sequenceName);
+            var sql = "SELECT NEXT VALUE FOR " + _sqlGenerator.DelimitIdentifier(_sequenceName);
```
Comment (anpete) [inline src/EntityFramework.SqlServer/SqlServerSequenceValueGenerator.cs]: Can this be added to the SQL generator so that it can be tested?

**Comment by anpete** [comment]
Doesn't look like there is a test for this change?

File: src/EntityFramework.SqlServer/SqlServerSequenceValueGenerator.cs (hunk 1 above)
Comment (mikary) [inline src/EntityFramework.SqlServer/SqlServerSequenceValueGenerator.cs]: Are you thinking something like ```AppendSelectValue(StringBuilder)```?

Comment (anpete) [inline src/EntityFramework.SqlServer/SqlServerSequenceValueGenerator.cs]: "Generate" is probably enough here, but yes, just so you can write a test for the change.

**Comment by mikary** [comment]
Updated based on PR feedback, I added the method to the SqlGenerator, but the name I came up with, ``` GenerateNextSequenceValueOperation``` seems a bit too verbose to me.

**Pull request merged**